Trappist/src/Promact.Trappist.Core/Controllers/TestConductController.cs

```diff
@@ -1,22 +1,25 @@
-﻿using Microsoft.AspNetCore.Authorization;
+﻿using Microsoft.AspNetCore.Http;
 using Microsoft.AspNetCore.Mvc;
+using Newtonsoft.Json;
 using Promact.Trappist.DomainModel.ApplicationClasses.TestConduct;
+using Promact.Trappist.DomainModel.Enum;
 using Promact.Trappist.DomainModel.Models.TestConduct;
 using Promact.Trappist.Repository.TestConduct;
 using Promact.Trappist.Repository.Tests;
+using Promact.Trappist.Utility.Constants;
+using System.Collections.Generic;
 using System.Threading.Tasks;
-using Newtonsoft.Json;
 
 namespace Promact.Trappist.Core.Controllers
 {
-    [Authorize]
     [Route("api/conduct")]
     public class TestConductController : Controller
     {
         #region Private Variables
         #region Dependencies
         private readonly ITestConductRepository _testConductRepository;
         private readonly ITestsRepository _testRepository;
+        private StringConstants constants;        
```

Comment:
Please use an injectable instance of StringConstants here. Do not use Concrete type.